### DISCUSSION TITLE
V1.1.0

### DISCUSSION THREAD

**Discussion by Joalor64** [Joalor64GH/Chocolate-Engine#4] [Announcements]
Another epic release (or something like that)!

Of course, this adds options lol.

Enjoy!

<hr /><em>This discussion was created from the release <a href='https://github.com/Joalor64GH/Chocolate-Engine/releases/tag/v1.1.0'>V1.1.0</a>.</em>

**Comment by Avrone1013** [comment]


**Reply by moxie-coder** [reply]
hey buddy, you have a problem with that, I’ll lock this discussion up
